site/en/blog/colrv1-fonts/index.md

```diff
@@ -0,0 +1,301 @@
+<figcaption>
+Material two-tone icons from <a href="https://fonts.google.com/icons">https://fonts.google.com/icons</a>
+</figcaption>
+</figure>
+
+### Artistic Expression
+
+Space-efficient color fonts enable new forms of artistic expression in text on
+the web. This example of a Kufi-style Arabic font uses color gradients as an
+artistic interpretation of what the ink flow of traditional calligraphy could
+look like when applied to the Kufi style of Arabic writing, which originates
+from not being written with nib and ink but carved in stone.
+
+<figure>{% Img
+src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/ZfUBAXQMogmAOsLwjcxO.png", alt="Arabic
+letters with gradients from black to red.", width="600", height="218" %}
+<figcaption><a href="https://github.com/aliftype/reem-kufi">Reem Kufi Ink</a>,
+an Arabic font by Khaled Hosny</figcaption> </figure>
+
+## What’s next?
+
+### Feature Detection
+
+At the moment, figuring out whether a browser engine supports a specific color
+font format is possible by means of user-agent sniffing or by searching in a
+library like [ChromaCheck](https://github.com/RoelN/ChromaCheck) by
+[@PixelAmbacht](https://twitter.com/pixelambacht) to test rendering of color
+glyphs on Canvas. Both solutions are not optimal. Feature testing should detect
+only a specific feature itself and avoid user agent sniffing. The ChromaCheck
+library should not need to perform resource-intensive 2D canvas operations for
+determining support.
+
+The Chrome team wants to improve that and has started a series of discussions
+[[1](https://github.com/w3c/csswg-drafts/issues/6520),
+[2](https://github.com/w3c/csswg-drafts/issues/6791)] in the CSS working group
+to provide information on browser font technology support in JS and
+declaratively in CSS. The team plans to release efficient feature detection for
+color font and other font technologies in a future version of Chrome.
+
+If you would like to use color fonts in your project right now when COLRv1
+support is limited to Chrome, there are two ways you can do that: Ask your font
+vendor for a COLRv1 font that contains monochrome glyphs too. User agents that
+don’t support COLRv1 will fall back to rendering monochrome
+glyphs. Alternatively, you can use the ChromaCheck library or user agent
+sniffing to determine whether COLRv1 support is available. Then you deliver CSS
+that loads COLRv1 fonts in supporting user agents and use an alternative font
+format such as COLRv0, a bitmap font format or OpenType SVG in other browsers.
+
+### CSS Font-Palette Support
+
+It would be tremendously useful if using a different set of colors would not
+require a new font. Thankfully, a mechanism exists, the
+[font-palette CSS property](https://www.w3.org/TR/css-fonts-4/#font-palette-prop). The
+Chrome team is working on adding
+[support for font-palette in Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=1170794).
+
+## COLRv1 Fonts & You!
```

Comment:
```suggestion
## COLRv1 fonts and you
```

Reply:
It's a common misconception that exclamation points are for adding emphasis. The clue that they're not is in the name; Stop! Wow! OMG!

Reply:
Ok, how about the exclamation point further down:

> With Chrome 98, we’re excited about how COLRv1 brings a whole new level of
typographic creativity to the web!

Remove that, too?

Reply:
That one should go. I didn't mention it because people to take it out of sentences is something of a losing game of whack-a-mole. I picked my battle and chose to keep it out of headings.